BASE/src/Microsoft.ApplicationInsights/Constants.cs

```diff
@@ -4,9 +4,7 @@ internal class Constants
     {
         internal const string ProfileQueryEndpoint = "https://dc.services.visualstudio.com/api/profiles/{0}/appId";
 
-        internal const string TelemetryNamePrefix = "Microsoft.ApplicationInsights.";
-
-        internal const string DevModeTelemetryNamePrefix = "Microsoft.ApplicationInsights.Dev.";
+        internal const string DevModeTelemetryNamePrefix = "Dev.";
```

Comment:
I don't think this logic is needed. It was introduced for Vortex customers and they will need to build a telemetry initializer now anyway. So let's just remove this extra thing from the base SDK.

Reply:
@SergeyKanzhelev do you have a contact on that team? I think we should communicate this change.

Reply:
It's all Vortex users. I don't have contacts, but Tillman should know